src/pages/Homepage.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Header from '../components/Header';
+import Sidebar from "../components/Sidebar";
+import Main from "../components/Main";
+
+function Homepage() {
+  return (
+    <div>
+      <Header/>
+
+      <div className="flex flex-col sm:flex-row-reverse items-start h-screen pt-10 min-w-[456px] ml-4 mr-4">
```

Comment:
`Body` was removed and I left the surrounding `div` here as is because i did not see the need to make this entire piece of code its own component.